src/state/psarstake/multiChainsHooks.ts

```diff
@@ -11,6 +11,10 @@ export type useDerivativeSarStakeType = {
   [chainId in ChainId]: typeof useDerivativeSarStake;
 };
 
+export type useDerivativeSarUnstakeType = {
+  [chainId in ChainId]: typeof useDerivativeSarUnstake;
+};
```

Comment:
- **please define all various hook type**
- **please do same for `useDerivativeSarStakeType` & `useSarPositionsType`**
```suggestion
export type useDerivativeSarUnstakeType = {
  [chainId in ChainId]: typeof useDerivativeSarUnstake | typeof useDerivativeHederaSarUnstake;
};
```